### ISSUE TITLE
snippet cannot effectively automatically

### ISSUE THREAD

**Issue by Meigumikato** [L3MON4D3/LuaSnip#329]
snippet c)
	Copyright `&enc[:2] == "utf" ? "©" : "(c)"` `strftime("%Y")` ${1:`g:snips_author`}. All Rights Reserved.

This snippet cannot effectively automatically replace some of the placeholders, such as encoding, time

**Comment by L3MON4D3** [comment]
That's correct, interpolation is not yet implemented for snipmate-like snippets.
I'll close this issue though, there already is #283 for this problem.
